python/mxnet/optimizer.py

```diff
@@ -798,9 +820,6 @@ class Ftrl(Optimizer):
         The initial learning rate.
     beta : float, optional
         Per-coordinate learning rate correlation parameter.
-    eta :
-        .. math::
```

Comment:
We should keep the clarification for eta chosen for this updater. Showing it as one of the parameters is not right place, probably just move it to line 791

Reply:
Should the last part of the math be `sqrt{\\sum_{s=1}^tg_{s,i}^2}}` instead?  @moveforever

Reply:
yes, it is. The eta is actually per-coordinate learning rate,. The equation should be moved to line 791